Automattic/woocommerce-payments
# Change read more link for warning message on Apple Pay domain registration failure

Fixes #4511

#### Changes proposed in this Pull Request
When Apple Pay domain verification fails, an error notification is shown with a "Learn more" link that points to the docs. We need to change the docs link, to point to the domain registration steps ( https://woocommerce.com/document/woocommerce-payments/payment-methods/apple-pay/#domain-registration ) 








#### Testing instructions




In order to register the domain to Apple Pay, server must use the WCPay live restricted keys for Apple Pay. We can use the account `acct_...ZoSU` . You can find the account here: paJDYF-18Y-p2#comment-5116.
Dev mode must be disabled in WCPay dev utils.
In order to trigger the domain registration, we must simulate that the plugin was just updated.
* change the content of the `apple-developer-merchantid-domain-association` 
* run the following SQL to simulate that you are on an older version: `UPDATE wp_options SET option_value='3.1.0' WHERE option_name='woocommerce_woocommerce_payments_version';`
* Refresh the admin page. You should see the following error:
<img width="1721" alt="Screenshot 2023-01-11 at 16 17 53" src="https://user-images.githubusercontent.com/82826872/211830467-c7bbdcd6-a82f-4d27-94d2-7792363b0fb5.png">
* Check the "Learn more" link
* Refresh the page again. You should see the following error:
<img width="1720" alt="Screenshot 2023-01-11 at 16 18 08" src="https://user-images.githubusercontent.com/82826872/211830419-25491e12-d8bc-459a-9575-48dcc99aec1b.png">
* Check the "Learn more" link
* If you want to run the test again, change the `docker/wordpress/.well-known/apple-developer-merchantid-domain-association` with its original value in order to be different from the one from root directory.

-------------------


**Post merge**
